coriolis/providers/replicator.py

```diff
@@ -519,10 +535,10 @@ def _copy_file(self, ssh, localPath, remotePath):
         except IOError as ex:
             if ex.errno != errno.ENOENT:
                 raise
+        finally:
             sftp.put(localPath, tmp)
```

Comment:
Please make double-sure that `sftp.put()` doesn't error if the file already exists

Reply:
It doesn't